server/db.js

```diff
@@ -374,8 +374,9 @@ exports.getQuestionsListByUser = function(request, callback) {
 
             for (q in docs) {
                 docs[q].firstAnswer = docs[q].answered[0] ? docs[q].answered[0] : 'No One';
-                docs[q].attemptsCount = docs[q].attempted.length;
+                docs[q].attemptedCount = docs[q].attempted.length;
```

Comment:
once the other PR is merged, everything down from here wont be included in the this PR